doc/building.md

```diff
@@ -1,4 +1,57 @@
-Building nyan
+Building the project
 =============
 
-no code yet :)
+Dependencies
+-------------
+
+```
+gcc >= 5.3.0 or clang >= 3.4
```

Comment:
it should be possible for "any" c++14 compliant compiler, so gcc 5.1 might do it as well.